tests/test_add_model.py

```diff
@@ -0,0 +1,423 @@
+from unittest.mock import MagicMock, Mock, patch
+
+import pytest
+from flask_restful import fields
+
+from flask_restful_swagger import swagger
+
+###############################################################################
+# Copy setup objects from examples/basic.py
+###############################################################################
+
+
+class MockBasicObject:
+    pass
+
+
+class MockBasicWithSwaggerMetadata1:
+    swagger_metadata = {"an_enum": {"enum": ["one", "two", "three"]}}
+
+class MockBasicWithSwaggerMetadata2:
+    def __init__(self, arg1, an_enum):
+        pass
+
+    swagger_metadata = {"an_enum": {"enum": ["one", "two", "three"]}}
+
+
+class TodoItem:
```

Comment:
Can we rename this to MockTodoItem or something similar to clarify that it's for testing?

Reply:
Renamed to `MockTodoItem`